mmdet/models/roi_heads/bbox_heads/bbox_head.py

```diff
@@ -108,8 +108,12 @@ def custom_accuracy(self):
     @auto_fp16()
     def forward(self, x):
         if self.with_avg_pool:
-            x = self.avg_pool(x)
-        x = x.view(x.size(0), -1)
+            if x.shape[0] > 0:
+                x = self.avg_pool(x)
+                x = x.view(x.size(0), -1)
+            else:
+                # Empty proposal
+                x = torch.mean(x, dim=(-1, -2))
```

Comment:
Why here does not be tested

Reply:
Because `self.with_avg_pool` is only set to True for certain algorithms, unit tests are not added.